### ISSUE TITLE
Error: Failed to lookup view "./index" in views directory "/Users/loh/dev/meme/built/web/"

### ISSUE THREAD

**Issue by benloh** [Vertex-Labs/meme#3]
As of dev-ds/package-updates-ds01 aae84af3ef49454278096f1b1c5ce236e66929dd, on an Intel Ventura 13.6.6 machine, running the following...

If we continue with the error after #2 and just run...
```
npm ci
npm run dev
```
...then pull up `http://localhost:3000`, we get the following error:
```
UR_EXPRESS   - GET / .
GET / 500 7.952 ms - 4565
Error: Failed to lookup view "./index" in views directory "/Users/loh/dev/meme/built/web/"
    at Function.render (/Users/loh/dev/meme/node_modules/express/lib/application.js:597:17)
    at ServerResponse.render (/Users/loh/dev/meme/node_modules/express/lib/response.js:1039:7)
    at /Users/loh/dev/meme/src/system/server-express.js:181:11
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/loh/dev/meme/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at /Users/loh/dev/meme/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:280:10)
    at /Users/loh/dev/meme/src/system/server-express.js:171:5
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:328:13)
    at /Users/loh/dev/meme/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:280:10)
    at cookieParser (/Users/loh/dev/meme/node_modules/cookie-parser/index.js:46:14)
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:328:13)
    at /Users/loh/dev/meme/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:280:10)
    at middleware (/Users/loh/dev/meme/node_modules/webpack-hot-middleware/middleware.js:46:48)
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:328:13)
    at /Users/loh/dev/meme/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:280:10)
    at goNext (/Users/loh/dev/meme/node_modules/webpack-dev-middleware/dist/middleware.js:60:16)
    at processRequest (/Users/loh/dev/meme/node_modules/webpack-dev-middleware/dist/middleware.js:76:15)
    at ready (/Users/loh/dev/meme/node_modules/webpack-dev-middleware/dist/utils/ready.js:16:5)
    at middleware (/Users/loh/dev/meme/node_modules/webpack-dev-middleware/dist/middleware.js:57:5)
    at Layer.handle [as handle_request] (/Users/loh/dev/meme/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:328:13)
    at /Users/loh/dev/meme/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/loh/dev/meme/node_modules/express/lib/router/index.js:280:10)
    at SendStream.error (/Users/loh/dev/meme/node_modules/serve-static/index.js:121:7)
    at SendStream.emit (node:events:517:28)
    at SendStream.error (/Users/loh/dev/meme/node_modules/send/index.js:270:17)
    at SendStream.onStatError (/Users/loh/dev/meme/node_modules/send/index.js:417:12)
    at onstat (/Users/loh/dev/meme/node_modules/send/index.js:722:26)
    at FSReqCallback.oncomplete (node:fs:202:21)
```

**Comment by benloh** [comment]
Won't fix.  Wrong branch.  Should have been `dev-vertex/packages-updated`
